app/scripts/controllers/network/network-controller.test.js

```diff
@@ -132,28 +96,65 @@ const JSONRPC_RESPONSE_BODY_PROPERTIES = ['id', 'jsonrpc', 'result', 'error'];
  */
 const INFURA_NETWORKS = [
   {
-    nickname: 'Mainnet',
+    networkName: 'Mainnet',
```

Comment:
[nit] I personally think we should be moving towards `chainName`. I know this is just for the test descriptions, but might as well move towards it

Reply:
We don't really need this property, so I removed it to avoid confusion: [6594eeb](https://github.com/MetaMask/metamask-extension/pull/17556/commits/6594eeb0905ce19d0ec3b828f3e4dfe7f6b13083)